src/kactl/data-structures/UnionFind.h

```diff
@@ -1,4 +1,5 @@
 /**
+ * Name: DSU
```

Comment:
I would probably prefer to write out the whole acronym - especially on DSU, where there's no estabilished standard acronym; DSU vs DFU vs UF...

Reply:
I've settled on Union-find, since that's the filename.